pkg/master/master.go

```diff
@@ -96,6 +96,12 @@ const (
 	DefaultEtcdPathPrefix = "/registry"
 )
 
+var (
+	// allAPIVersions records all AVIVersions that are supported at /apis, in the
+	// form of "group/version"
+	allAPIVersions []string
```

Comment:
Don't use a global for this; at run time you should just get the list from Brendan's 3rd party resource controller thingy.